packages/plugin-infra/src/site-adaptor/PostContext.ts

```diff
@@ -22,6 +22,7 @@ export interface PostContextAuthor {
     readonly nickname: Subscription<string | null>
     readonly avatarURL: Subscription<URL | null>
     readonly author: Subscription<ProfileIdentifier | null>
+    readonly authorPublicKey: Subscription<EC_Key | null>
```

Comment:
```suggestion
    readonly authorPersona: Subscription<PersonaIdentifier | null>
```

you should convert `EC_Key` to `PersonaIdentifier`. `EC_Key` should only used in encryption.